packages/projection/src/operators/Mappers/withNftMetadata.ts

```diff
@@ -1,37 +1,31 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import { Asset, Cardano } from '@cardano-sdk/core';
 import { WithCIP67 } from './withCIP67';
 import { WithMint } from './withMint';
 import { WithUtxo } from './withUtxo';
-import { dummyLogger } from 'ts-log';
 import { isNotNil } from '@cardano-sdk/util';
+import { logger } from '@cardano-sdk/util-dev';
```

Comment:
`util-dev` package should only be used for tests. Please inject a `Logger` in the argument to `withNftMetadata`